### DISCUSSION TITLE
Script for Kali Linux

### DISCUSSION THREAD

**Discussion by mciarciaglini** [tteck/Proxmox#2057] [Ideas]
Could you possibly write a script for Kali linux? 

Thanks!

**Comment by X-Anonymous-Y** [comment]
> Could you possibly write a script for Kali linux?
> 
> Thanks!

I dont think! because official supported list off  Proxmox: [Templates](https://pve.proxmox.com/wiki/Linux_Container#pct_supported_distributions)

**Comment by mciarciaglini** [comment]
Not sure what you mean.  It is an ubuntu based distro.

Thanks!
________________________________

**Reply by X-Anonymous-Y** [reply]
Sorry! my English is not so good.

Yes, it is correct, it is an Ubuntu based Distro but with four changes in the substructure and many security futures built in. You would now just have to find out what has changed in the substructure and what security measures have been introduced in order to be able to create an LXC container from it.
